src/main/java/nl/knaw/dans/ingest/core/ImportArea.java

```diff
@@ -83,6 +89,14 @@ public String startImport(Path inputPath, boolean isBatch, boolean continuePrevi
         return relativeInputDir.toString();
     }
 
+    private Boolean isSafeInputPath(Path inputPath) {
+        if ( inputPath.toAbsolutePath().normalize().startsWith(getInboxDir())) {
```

Comment:
Don't you need a normalize on the inbox too? Here or within the get method. Otherwise it is looking good to me.